src/pages/MyNotes/components/Note/Note.tsx

```diff
@@ -0,0 +1,36 @@
+import { FC } from 'react';
+import { Button, CardContent, Typography } from '@mui/material';
+import { Delete as DeleteIcon } from '@mui/icons-material';
+
+import { EditMenu, ShareButton } from 'components';
+import { BUTTON_TEXT } from 'config/types';
+import { INoteProps } from 'pages/MyNotes/types';
+import { getShortDescription } from 'utils';
+
+import {
+  StyledCard,
+  StyledTitle,
+  StyledDateCreation,
+  StyledCardActions,
+} from './styles';
+
+const Note: FC<INoteProps> = ({ note, handleActiveNote, editNote }) => (
+  <StyledCard>
+    <ShareButton note={note} />
+    <CardContent onClick={handleActiveNote}>
+      <StyledTitle variant='h3'>{note.title}</StyledTitle>
```

Comment:
please desctructure all title, dateCreation, ... from note before return statement and reuse here as title, dateCreation, ...